File: Data Structures & Algorithms/design-hashmap/submission-0.go
type MyHashMap struct {
	data [][2]int
}

func Constructor() MyHashMap {
    return MyHashMap{data: [][2]int{}}
}

func (this *MyHashMap) Put(key int, value int) {
    for i, duo := range this.data {
		if duo[0] == key {
			this.data[i][1] = value
			return
		}
	}

	this.data = append(this.data, [2]int{key, value})
}

func (this *MyHashMap) Get(key int) int {
    for _, duo := range this.data {
		if duo[0] == key {
			return duo[1]
		}
	}

	return -1
}

func (this *MyHashMap) Remove(key int) {
    for i, duo := range this.data {
		if duo[0] == key {
			this.data = append(this.data[:i], this.data[i+1:]...)
			return
		}
	}
}

/**
 * Your MyHashMap object will be instantiated and called as such:
 * obj := Constructor();
 * obj.Put(key,value);
 * param_2 := obj.Get(key);
 * obj.Remove(key);
 */

File: Data Structures & Algorithms/implement-queue-using-stacks/submission-0.py
class MyQueue:

    def __init__(self):
        self.stack = []
        

    def push(self, x: int) -> None:
        self.stack.append(x)
        

    def pop(self) -> int:
        stack_temp = []

        length = len(self.stack)
        for i in range(length - 1):
            stack_temp.append(self.stack.pop(-1))

        ans = self.stack.pop(-1)

        for i in range(length - 1):
            self.stack.append(stack_temp.pop(-1))

        return ans
        

    def peek(self) -> int:
        stack_temp = []

        length = len(self.stack)
        for i in range(length):
            stack_temp.append(self.stack.pop(-1))

        ans = stack_temp[-1]

        for i in range(length):
            self.stack.append(stack_temp.pop(-1))

        return ans
        

    def empty(self) -> bool:
        return len(self.stack) == 0
        


# Your MyQueue object will be instantiated and called as such:
# obj = MyQueue()
# obj.push(x)
# param_2 = obj.pop()
# param_3 = obj.peek()
# param_4 = obj.empty()